### PR TITLE
added NCDS'17 summit

### PR DESCRIPTION
**Pull request by vyaspranjal33**
**Details of the conference**:

- Website:
- Date:
- Venue:
- Description:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ If you would like to be notified of future conferences that would be added to th
 
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
+|[National Cyber Defence](https://summit.ncdrc.res.in/) |15-16 Dec | Goa | National Cyber Defence Summit is Asia's Biggest Cyber conclave dedicated to taking Cyber doamin of India to next level
```
Comment (kspviswa) [inline README.md]: daomin => domain

Comment (nikhita) [reply README.md]: Thanks for the review, @kspviswa!

**Comment by nikhita** [comment]
@vyaspranjal33 A minor note: in case you decide to contribute in the future (hopefully you do! :smile:), please mention the conference details in the Pull Request body: https://github.com/nikhita/tech-conferences-india/pull/51#issue-274469713.

**Comment by vyaspranjal33** [comment]
u mean description of PR ?
@nikhita

**Comment by nikhita** [comment]
> u mean description of PR ?

Yes :)

**Comment by vyaspranjal33** [comment]
Ok will take care :)

**Pull request merged**